packages/kbn-dev-utils/src/run/run_with_commands.ts

```diff
@@ -122,11 +125,14 @@ export class RunWithCommands<T> {
         await command.run(extendedContext);
       });
     } catch (error) {
+      await metrics.reportError(commandName);
```

Comment:
I think we should cleanup first personally

Reply:
165286dfa471aa6f21dbadf949a1ce550975441a